src/Message/Topics.php

```diff
@@ -137,9 +137,25 @@ public function nest(Closure $callback, $condition)
 
         $callback($topic);
         if (count($topic->conditions)) {
+            $needsBackwardCompatibility = collect()
+                ->push(" || ", " && ")
+                ->contains(null, $condition);
+
+            if (
+                !$needsBackwardCompatibility &&
+                !collect()
+                    ->push("||", "&&")
+                    ->contains(null, $condition)
+            ) {
+                throw new \Exception(
+                    "You must use either one of (|| , &&) as a condition"
+                );
+            }
```

Comment:
Could this be written in a more PHP pure solution ?

Thanks for the tests !

Reply:
Sure I'll send a pure implementation soon, no problem at all!

Reply:
Done here : https://github.com/code-lts/Laravel-FCM/pull/17/commits/eb001e7695b00a62878dee775843a0d507518e2c

The same tests above still work.